spec/system/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward', type: :system do
```

Comment:
I think it was supposed to be something like this:
```suggestion
RSpec.describe 'Order', type: :system do
```